engine/api/workflow/run_workflow_test.go

```diff
@@ -410,7 +412,7 @@ func TestManualRun3(t *testing.T) {
 		}
 
 		//TestUpdateNodeJobRunStatus
-		_, err = workflow.UpdateNodeJobRunStatus(nil, db, db, cache, proj, j, sdk.StatusSuccess)
+		_, err = workflow.UpdateNodeJobRunStatus(nil, func() *gorp.DbMap { return db }, db, cache, proj, j, sdk.StatusSuccess)
```

Comment:
do not pass a nil Context, even if a function permits it; pass context.TODO if you are unsure about which Context to use